ui/app/components/ui/page-container/index.scss

```diff
@@ -98,17 +98,17 @@
   }
 
   &__title {
+    @include H2;
+
     color: $black;
-    font-size: 2rem;
     font-weight: 500;
-    line-height: 2rem;
     margin-right: 1.5rem;
   }
 
   &__subtitle {
+    @include H6;
```

Comment:
This is different by .1rem but I don't see this subtitle being used anywhere except the gas-modal which is then hidden using css as well <_<